websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,106 @@
+const presence = new Presence({ clientId: "937015924425367643" }),
+	startTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			details: "Geziniyor",
+			startTimestamp,
+		},
+		{ pathname, search } = document.location,
+		searchParams = new URLSearchParams(search);
+
+	switch (pathname) {
+		case "/":
+			presenceData.details = "Anasayfayı görüntülüyor";
+			break;
+		case "/yetkililer":
+			presenceData.details = "Yetkililer sayfasını görüntülüyor";
+			break;
+		case "/share":
+			presenceData.details = "Kod paylaşıyor";
+			break;
+		case "/tools":
+			presenceData.details = "Araçlar sayfasını görüntülüyor";
+			break;
+
+		case "/profile":
+			presenceData.details = "Profil görüntüleniyor:";
+			presenceData.state = document.querySelector(".user-info").textContent;
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>("img.avatar").src;
+			break;
```

Comment:
Are you sure this works? This will only trigger if the pathname is strictly equal to /profile, not if it includes that.

Reply:
sure
![image](https://user-images.githubusercontent.com/55600314/174436701-ff762730-df7b-4f48-96b5-5f4803004d59.png)
![image](https://user-images.githubusercontent.com/55600314/174436704-2c41ba4c-bf3a-411d-ac0b-2248e65795b9.png)

Reply:
Interesting